[fireeye] Add counters for requests and vulnerabilities

n1chre opened the pull request:
Adds counters to the program for all kinds of stuff.

skogtwin commented on cmd/fireeye2nvd/fireeye2nvd.go:
```diff
@@ -55,18 +56,21 @@ func main() {
 		dur, err := time.ParseDuration("-" + *sinceDuration)
 		if err != nil {
-			log.Fatalln(err)
+			log.Println(err)
```
We loose stacktrace with this change... What about that idea of yours to defer the `recover()` call?

skogtwin commented on cmd/fireeye2nvd/fireeye2nvd.go:
```diff
@@ -55,18 +56,21 @@ func main() {
 	sinceUnix := flag.Int64("since_unix", 0, "Unix timestamp since when should we download. If not set, downloads all available data")
 	sinceDuration := flag.String("since", "", "Golang duration string, overrides -since_unix flag")
 	dontConvert := flag.Bool("dont_convert", false, "Should the feed be converted to NVD format or not")
+	stats.Global.AddFlags()
```
This additional level of indirection is kind of ugly, why not just `stats.AddFlags()`, `stats.WriteAndLogError()` etc.?

n1chre commented:
un-exported global stats and added functions that operate on it
The reason why I removed `Fatalln` is because it calls os.Exit(1) internally so defer, recover - don't work. log.Panicln() would work, but Println is also ok since it's only in the main file.
